src/navigation_controller.ts

```diff
@@ -69,37 +68,36 @@ export class NavigationController {
   /**
    * Removes methods on core Blockly components that allows them to handle
    * keyboard shortcuts.
-   * @protected
    */
-  removeShortcutHandlers() {
+  protected removeShortcutHandlers() {
     if (Blockly.Toolbox) {
-      Blockly.Toolbox.prototype.onShortcut = null;
+      Blockly.Toolbox.prototype.onShortcut = () => false;
     }
   }
 
   /**
    * Handles the given keyboard shortcut.
    * This is only triggered when keyboard accessibility mode is enabled.
-   * @param {!ShortcutRegistry.KeyboardShortcut} shortcut The shortcut
-   *     to be handled.
-   * @returns {boolean} True if the toolbox handled the shortcut,
-   *     false otherwise.
-   * @this {Blockly.Toolbox}
-   * @protected
+   *
+   * @param shortcut The shortcut to be handled.
+   * @returns True if the toolbox handled the shortcut, false otherwise.
    */
-  toolboxHandler(shortcut) {
+  protected toolboxHandler(
+    this: Blockly.Toolbox,
+    shortcut: ShortcutRegistry.KeyboardShortcut,
+  ): boolean {
     if (!this.selectedItem_) {
       return false;
     }
     switch (shortcut.name) {
       case Constants.SHORTCUT_NAMES.PREVIOUS:
-        return this.selectPrevious_();
+        return (this as any).selectPrevious_();
```

Comment:
Might have missed this in previous PRs, but do you want to set up an `AnyDuringMigration` alias? Not sure if there are places we actually want to be using `any` vs places where we're just doing it temporarily.

Reply:
The linter warns about uses of `any` so I'm going to rely on that instead of an `AnyDuringMigration` until I'm convinced that we have legitimate uses of `any` instead of just patching over issues.